README.md

```diff
@@ -3,6 +3,14 @@ GOV.UK Frontend ·
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 =====================
 
+> **Warning**
+>
+> We’re currently implementing extensive changes to our code for [our next major version](https://github.com/alphagov/govuk-frontend/milestone/46).
+>
+> While we're making changes, only fork this repository if necessary. This is to limit how often you will need to update your fork to keep up to date.
+>
+> If you're fixing a bug in our previous releases, branch off the [latest tag](https://github.com/alphagov/govuk-frontend/tags) instead.
```

Comment:
Looks grand. One small thought - is there a possibility of the "latest tag" at some point being something like "5.0.alpha_pre_release" or whatever?

We haven't discussed anything like that, but something to keep in mind perhaps if we go that route.

Reply:
I'll clarify to "latest 4.x tag", that's a good shout 😊